### PR TITLE
BS4 General Fixes

### LINKED ISSUE #3085: BS4 Fixes Needed
### Describe the bug

After multiple revisions to the encounter tab, I believe there still are improvements that need to be addressed here. Also some BS4 code fixes are needed.

Current Issues Found:
- Encounter top isn't using BS4 colors (also it looks like it needs a little padding).
- Encounter top isn't sizing correctly and is showing a scrollbar on my end
- The encounter page has buttons that aren't sized correctly in it
- It seems like we need to add more colors to the encounter page as it is looking kinda bland
- There isn't any body padding in the encounter page for some reason
- BS4 fixes are needed from prior commits

I'll be fixing these in a pull request @bradymiller @sjpadgett please assign me.

### LINKED ISSUE #3177: All oe-styles are missing BS4 colors in colors array
### Describe the bug

In all themes located in `/interace/themes/oe-styles`, there should be in the `$colors: (` array the following values:
- `"gray600": $gray-600`
- `"gray800": $gray-800`

Apparently I forgot to add these values to the array. They are required for themes to work correctly.

### PR DESCRIPTION
**Pull request by tywrenn**
Fixes #3085
Fixes #3177 

- Brings in general fixes to a ton of files
- Fixes missing array colors in themes
- Replaces all that I could find that needed replacing to comply with BS4

### CODE REVIEW

File: interface/drugs/drug_inventory.php [hunk 1]
```diff
@@ -134,31 +134,33 @@ function mapToTable($row)
 <title><?php echo xlt('Drug Inventory'); ?></title>
 
 <style>
-a, a:visited, a:hover { color:#0000cc; }
+a, a:visited, a:hover {
```
Comment (tywrenn) [inline interface/drugs/drug_inventory.php]: @bradymiller why do we have this set here?

**Comment by tywrenn** [comment]
@bradymiller Seems like Travis is failing on every commit again lol

File: interface/drugs/drug_inventory.php (hunk 1 above)
Comment (bradymiller) [reply interface/drugs/drug_inventory.php]: not sure. looks like it has been in this script since the beginning of time though.

**Comment by bradymiller** [comment]
Looks like travis fail is from:
```
PHP Parse error:  syntax error, unexpected 'align' (T_STRING), expecting ',' or ';' in ./interface/drugs/add_edit_drug.php on line 455
```

**Comment by bradymiller** [comment]
hi @tywrenn , Code looks good to me. If testing well for you (and the above parse error is fixed), then let me know, and will bring it into codebase.

**Comment by tywrenn** [comment]
@bradymiller Still checking for more fixes -- will tell you when i'm ready to merge

**Comment by tywrenn** [comment]
@bradymiller Ready for testing

**Comment by bradymiller** [comment]
hi @tywrenn , you've got some conflicts to manage. rec that your address those as soon as possible and then getting this into codebase to avoid more conflicts; ie. codebase is moving fast.

**Comment by tywrenn** [comment]
@bradymiller Just fixed merge conflicts within GitHub and merged master (that's a cool feature!)

**Comment by bradymiller** [comment]
hi @tywrenn , Looks good and bringing into codebase. Thanks for the imrpovements! -brady

**Pull request merged**